app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -81,9 +85,9 @@ show_navigation_menu_text = t("components.layout_super_navigation_header.menu_to
             li_classes = %w[gem-c-layout-super-navigation-header__navigation-item]
             li_classes << "gem-c-layout-super-navigation-header__navigation-item--with-children" if has_children
             unique_id = SecureRandom.hex(4)
-
             show_menu_text = t("components.layout_super_navigation_header.menu_toggle_label.show", :label => link[:label])
             hide_menu_text = t("components.layout_super_navigation_header.menu_toggle_label.hide", :label => link[:label])
+            tracking_label = link[:label].downcase.gsub(/\s+/, "")
           %>
           <%= tag.li class: li_classes do %>
             <a class="gem-c-layout-super-navigation-header__navigation-item-link" href="<%= link[:href] %>">
```

Comment:
Does this link need anything extra so that clicks on it can be tracked?

Reply:
Ignore me, there's no requirement for these links to be tracked. Plus they're hidden when JavaScript is enabled, which would make tracking clicks on them impossible.